crates/chain-config/src/config/snapshot_metadata.rs

```diff
@@ -19,6 +19,7 @@ pub enum TableEncoding {
     },
 }
 impl TableEncoding {
+    #[allow(clippy::assigning_clones)] // TODO: clean up the following code
```

Comment:
Let's create an issue for this.

Reply:
https://github.com/FuelLabs/fuel-core/issues/2100